Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -445,9 +445,8 @@
     (global-name "com.apple.lsd.mapdb")) 
 
 ;; <rdar://problem/12413942>
-(with-filter (require-not (state-flag "BlockMobileGestaltInWebContentSandbox"))
-    (allow file-read*
-       (well-known-system-group-container-literal "/systemgroup.com.apple.mobilegestaltcache/Library/Caches/com.apple.MobileGestalt.plist")))
+(allow file-read*
```

Comment:
Does web process still need access to this file when MobileGestalt service is blocked?

Reply:
Yes, that is correct. We are only aiming to block the service, since the read access to the cache is not something we are concerned about. This specific change reverts the sandbox back to the state before the setting was added.

Thanks for reviewing!